bool    Gun_mode
bool    Fire
bool    Reload
float32 Turn
